sci-libs/calculix-ccx/calculix-ccx-2.10.ebuild

```diff
@@ -44,6 +48,11 @@ src_configure() {
 	if use threads; then
 		append-cflags "-DUSE_MT"
 	fi
+	
+	if use openmp; then
+		export FFLAGS="-fopenmp $(FFLAGS)"
```

Comment:
`$()` in ```bash``` executes the content in a subshell.

Simply use the `append-fflags` function from `flag-o-matic.eclass`